docs/runtime/nodejs-metrics.md

```diff
@@ -313,10 +313,10 @@ This metric is [recommended][MetricRecommended].
 
 | Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
 | -------- | --------------- | ----------- | -------------- | --------- |
-| `nodejs.eventloop.utilization` | Gauge | `1` | Event loop utilization. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `nodejs.eventloop.utilization` | Counter | `s` | Event loop utilization. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
nodejs perf_hooks expose 2 cumulative counters to measure utilization:
https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2

active seconds and idle seconds

the utilization value itself seems to somehow be computed based on the difference between two values and it is not clear if that could be represented as a counter directly.

my suggestion would be to add 
```
nodejs.eventloop.active.time
nodejs.eventloop.idle.time
```
and then keep utilization as a gauge if we think it is useful.

Reply:
Created a new metric for time, with the state as an attribute. Kept the utilization as is.